script/Ciel/Day 9/09_01_CIEL09_2B.txt

```diff
@@ -422,113 +480,126 @@ This cycle has continued for a long, long time."
+"Well... I guess you could say that.
+ I was just wondering what sort of vampire would come up with such a roundabout solution to immortality..."
 }
 [sha:a8c489a579e1b7120e4fa22e6b22cab1cb7e83dc]{
 -- Page 187, Offset 22601.
 -- 「そうでしたか。ではもう少しだけ。
 // Ahhh......  ahhhhhhhhhhh.............
-"Is that so? Well, I'll tell you a bit more about him then.
+// 'もう少しだけ'? [Checks statusline] We aren't even 1/3 of the way in
+"Is that so? Alright, I'll tell you a little more about him.
 }
 [sha:5815279a13ed2dfc55528cb847839e20b1e56cea]{
 -- Page 187, Offset 22602.
 -- 始まりのロアは男性ですが、その性別は転生先の肉体によって変化します。
-Roa was originally a man, but his gender changes in accordance with his host body.
+Roa was originally a man, but his sex changes in accordance with his host body.
 }
 [sha:c80dbc8591e5a4554a3501d2d3adf7b825984b20]{
 -- Page 187, Offset 22603.
 -- この死徒の厄介なところは、その発見が極めて困難という点ですね。なにしろ人の子として生まれてきて、きちんとした両親がいるんです」
-It's the fact that detecting him is extremely difficult that makes him such a pain to deal with. He is, after all, born as a human child with proper parents."
+// This section very similar to Arc 10_6 offsets 26237+, very cool that we get to rehash it again
+What makes him a particularly troublesome Dead Apostle to deal with is that he's such a pain to find.
+ He is, after all, born as a human child to ordinary human parents."
 }
 [sha:8a21481f4690ec4d7aedc33e4dee8a4e54a312a8]{
 -- Page 188, Offset 22604.
 -- 「ロアが吸血鬼として肉体と意識を変貌させるのは、肉体が満足に活動できる歳からになります。
-"It's only after the body has developed adequately that the vampire Roa begins to alter their body and mind.
+"Roa will only begin to mold the body and mind of his host into that of a vampire after they reach the age where they can act freely.
 }
 [sha:a9ab6391b4b07922b8076d07e5caf4fafb98d28d]{
 -- Page 188, Offset 22605.
 -- ですのでロアの転生体は成長するまで吸血鬼としての片鱗をまったく見せません。そのくせ一度ロアとして目覚めたら、今まで培った人間関係を利用して完全に社会に融け込むんです」
-As such, until his host has matured, they will show absolutely no signs of vampirism. Because of this, once they have awakened, Roa can use the interpersonal relationships that the host has cultivated to blend perfectly into society."
+As such, until his host has matured, they will show absolutely no signs of vampirism.
+ A side effect of this is that even after he awakens,
+ Roa can continue to use his host's network of relationships to blend in seamlessly with society."
 }
 [sha:c4b0fbda36695e6760535bd5227389bf85ff3ad5]{
 -- Page 189, Offset 22606.
 -- 「教会がロアに気づいた時にはもう手遅れのケースがほとんどでした。ロアが育った街は死の都と化し、その絶頂期にアルクェイドによって終わらされています」
-"By the time the Church detects him, it's almost always too late. The city Roa was raised in is transformed into a city of the dead, and is felled at its peak by Arcueid."
+"The majority of the time the Church manages to detect him it's already too late.
+ The city Roa grew up in has already been transformed into a necropolis, reached its zenith, and been toppled by Arcueid."
```

Comment:
It feels better to me with this aspect imo - the church's poor track record is an ongoing thing, and we're talking about multiple cases here instead of just the one. I can make the zenith association more clear though.

Reply:
I think most important is that ロアが育った街は死の都と化し and その絶頂期にアルクェイドによって終わらされています need to be more seperated.